index.js

```diff
@@ -249,6 +290,12 @@ var FirefoxBrowser = function (id, baseBrowserDecorator, args) {
         // Ignore failure -- the browser process might have already been
         // terminated.
       }
+    } else if (isWsl) {
+      const tasklist = extractPids (safeExecSync ('tasklist.exe /FI "IMAGENAME eq firefox.exe" /FO CSV /NH /SVC'))
+        .filter(pid => browserProcessPidWsl.indexOf(pid) === -1)
+
+      log.warn ('Killing the following PIDs: %s', tasklist)
+      log.debug (safeExecSync ('taskkill.exe /F ' + tasklist.map (pid => `/PID ${pid}`).join (' ')))
```

Comment:
Nit: It seems odd to me to execute the code to kill the tasks as an argument to `log.debug()`. I think many people will read the code and skip over the `log.debug()` line assuming it is just for logging.

Can we separate this out into two lines? One that kills the tasks and records the result, and one that logs it?

Reply:
Sure